containersQa/testlib.bash

```diff
@@ -942,21 +942,24 @@ function setupAlgorithmTesting {
   cipherListCode=`cat $LIBCQA_SCRIPT_DIR/CipherList.java`
 }
 
-function logHostAndContainerCryptoPolicy() {
+function checkHostAndContainerCryptoPolicy() {
```

Comment:
I would vote to log as one step, and keep logs, and asser as second step.  they can both call same logic, jsut one with asserst on (and the os fs) and second without